core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -56,6 +58,21 @@ object ToResponse extends LowPriorityToResponseInstances {
     rep
   }
 
+  implicit def asyncStreamToResponse[A, CT <: String](implicit
```

Comment:
> In this case, we don't even need a new case for AsyncStream[A], we can generalize our previous version AsyncStream[Buf] to AsyncStream[A: Encode].

Yea, but to use EOL we need a special case for `AsyncStream[A]`. I did some refactoring